model/camera/camera.go

```diff
@@ -152,6 +154,45 @@ func (c *component) DoCommand(ctx context.Context, command map[string]interface{
 			"command": "fetch",
 			"video":   videoBytesBase64,
 		}, nil
+	case "get-storage-state":
+		c.logger.Debug("get-storage-state command received")
+		state, err := c.videostore.GetStorageState()
+		if err != nil {
+			return nil, err
+		}
+
+		vsConfig := c.videostore.GetConfig()
+		storagePath := vsConfig.Storage.StoragePath
+		storageLimitGB := vsConfig.Storage.SizeGB
+
+		fsUsage, err := diskusage.Statfs(storagePath)
```

Comment:
can we move this into `videostore.GetStorageState`?

I'd like DoCommand to be:
```
switch command {
case "command-string":
		state, err := c.videostore.GetStorageState()
		if err != nil {
			return nil, err
		}
                 return formatState(state), nil

...
}
```

Reply:
Yes it has been moved